src/components/Header.js

```diff
@@ -0,0 +1,51 @@
+import { Link, Routes, Route } from "react-router-dom";
+import Hawaii from "./Hawaii";
+import React from "react";
+import Sicily from "./Sicily";
+import Home from "./Home";
+
+const Header = ({ hotels, hawaiiSuites, sicilySuites }) => {
```

Comment:
interesting name choice. i found the function of it to be more than what i'm used to for headers. it seems to have more logic in it than a simple nav. but i see that you used routing and so maybe that's why? idk much about react so feel free to ignore my comment if it doesn't make sense